.eslintrc.js

```diff
@@ -106,6 +106,7 @@ module.exports = {
     'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
     'no-useless-escape': 'off',
     'no-console': 'warn',
+    'unicorn/no-fn-reference-in-iterator': 'off',
```

Comment:
My preference would be to leave this enabled. I've been bitten by this exact issue on Cumulus before, and it was a nightmare to try to figure out.

Reply:
This wasn't intentional.  👎

Reply:
8f05b5743